### PR TITLE
Postgres CDC

### PR DESCRIPTION
**Pull request by blinktag**

### CODE REVIEW

File: backends/cdc-postgres/cdc-postgres.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import (
+	"crypto/tls"
+	"github.com/batchcorp/plumber/cli"
+	"github.com/batchcorp/plumber/printer"
+	"github.com/jackc/pgx"
+	"github.com/sirupsen/logrus"
+)
+
+/*
+Example output
```
Comment (dselans) [inline backends/cdc-postgres/cdc-postgres.go:12]: Do we track the `xid` as well somewhere?

Comment (blinktag) [reply backends/cdc-postgres/cdc-postgres.go:12]: Whoops, added

File: backends/cdc-postgres/read.go [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+			changeRecord.Changes = append(changeRecord.Changes, change)
+		}
+		return nil
+	}
+	return sub.Start(context.Background(), 0, handler)
+}
+
+// validateReadOptions ensures the correct CLI options are specified for the read action
+func validateReadOptions(opts *cli.Options) error {
+	return nil
```
Comment (dselans) [inline backends/cdc-postgres/read.go:149]: Do we not have to validate any inputs?

File: backends/kafka/relay.go [hunk 3]
```diff
@@ -74,7 +74,7 @@ func Relay(opts *cli.Options) error {
 		Options:        opts,
 		RelayCh:        relayCfg.RelayCh,
 		log:            logrus.WithField("pkg", "kafka/relay"),
-		Looper:         director.NewFreeLooper(director.FOREVER, make(chan error)),
+		Looper:         director.NewFreeLooper(director.FOREVER, make(chan error, 1)),
```
Comment (dselans) [inline backends/kafka/relay.go:77]: Nice catch!

**Review by dselans — APPROVED** [review]
We should add some examples on how to use CDC w/ postgres + update README.md.

File: backends/cdc-postgres/read.go (hunk 2 above)
Comment (blinktag) [reply backends/cdc-postgres/read.go:149]: Nope! they are already required at the CLI flag level. There is currently no optional/conditional inputs

**Pull request merged**